recipes/bcgtree/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "bcgtree" %}
+{% set version = "1.2.1" %}
+{% set sha256 = "7739da7bf8e36933a45ee62e72cdf5988626595223baab96ec24a657a814b4a2" %}
+
+package:
+  name: '{{ name }}'
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/molbiodiv/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+  noarch: generic
+  run_exports:
+    - {{ pin_subpackage('bcgtree', max_pin="{{ version }}") }}
```

Comment:
```suggestion
    - {{ pin_subpackage('bcgtree', max_pin="x") }}
```

Reply:
See https://github.com/bioconda/bioconda-recipes/blob/master/.github/PULL_REQUEST_TEMPLATE.md#instructions-for-avoiding-api-abi-and-cli-breakage-issues